src/utils/constants.ts

```diff
@@ -73,6 +75,42 @@ const CONFIGURE_TABS: ConfigureTab[] = [
   { type: 'wallet', label: 'Wallet' },
 ]
 
+type UserLimit = {
+  isUnlimited: boolean
+  value?: number
+  unit?: 'MB' | 'GB'
+}
+
+const unlimitedUserLimit: UserLimit = {
+  isUnlimited: true,
+}
+
+const defaultUserLimit: UserLimit = {
+  isUnlimited: false,
+  value: 2,
+  unit: 'MB',
+}
+
+type UserLimitType = 'Unlimited' | 'Limited'
+
+const storageValues = ['MB', 'GB']
+
+type BandwidthLimitUnits = {
```

Comment:
Use singular names for types.

```suggestion
type BandwidthLimitUnit = {
```